src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -24,7 +24,7 @@ private Settings(IConfigurationSource source)
                 throw new ArgumentNullException(nameof(source));
             }
 
-            Exporter = source.GetString(ConfigurationKeys.Exporter) ?? "otlp";
+            TracesExporter = ParseTracesExporter(source);
```

Comment:
Idea for a separate PR: the code for OtlpExport settings should be only executed if Otlp is actually being used.